### DISCUSSION TITLE
开往征文活动第一期题目

### DISCUSSION THREAD

**Discussion by linlinzzo** [travellings-link/travellings#1078] [发展建议]
### 卷一

出卷：林林

水，最容易消失在水中；人，最容易消失在人群里。一滴水清晰可见，但是，当它融入水中，你还能找到那滴水在哪里吗？同样，你是一个有个性、有思想、有自我的人，如果迫切希望被别人接纳，渴望合群，常常会磨掉自己的棱角，隐藏自己的个性，屏蔽自己内心的声音。马克·吐温说：“跟世界上所有的人一样，我所暴露给世人的只是修剪过的、洒过香水的、精心美容过的公开的意见，而把我私底下的意见谨慎小心地、聪明地遮盖了起来。”因为，只有当你变得与人群中的其他人没有多大区别的时候，这个群体才会接纳你。——廖之坤

以上材料具有启示意义，请结合材料和生活经验，撰写一篇博文。

### 卷二

出卷：林林

请根据图片内容，结合自己的工作生活经验，撰写一篇博文。

（图片文字内容：误删了一大段代码后发现软件还能跑起来，这时候我们一定不要动他。图片来自网络）
![image](https://user-images.githubusercontent.com/79007796/178895801-5900cc0c-a206-4b3a-8cbf-881dae474c1e.png)

### 卷三

出卷：林林

霓虹闪亮，暮色苍茫。
寒风刺骨，冷月如霜。
白发的母亲还在倚门眺望，
年幼的儿女一次次把手机拨响，
桌上的饭菜散发出诱人的馨香……

为何不回家，你还在何方？
你在娇子一号……
人海茫茫把你淹没，
绵绵人流在你面前伸长……
几百次的操作，镇定自若哪管手僵。
几千次的重复，面不改色绝不慌张。
喊哑了嗓子，累弯了腰。
送走了月亮，迎来了太阳！
你在一寸一寸挤压病毒的领地，
你在一分一秒抢夺生命的曙光！
用滚烫的热血浇灌美丽的芙蓉，
以钢铁的脊梁扛起锦城的安康！

我们离去了，进入甜蜜的梦乡。
你依然目光炯炯，
在为人民站岗！
你来自哪里？
武侯、高新还有双流、青白江……

啊，我要歌唱
我要为造就你的崭新时代歌唱，
我要为滋养你的伟大祖国歌唱，
我要为培育你的伟大的中国共产党歌唱！
啊，歌唱、歌唱！
歌唱你这人间的天使，
歌唱你这平凡的英雄，
歌唱你这寒夜中温暖的阳光！
《寒夜中温暖的阳光》——献给为全民核酸检测辛勤工作的人们

请结合材料和生活经验，撰写一篇博文。

### 卷四

出卷：Jason

由于jsdelivr ICP备案被调销，请以优化网站速度为中心撰文，不限于优化jsd。

**Comment by essesoul** [comment]
emmm？？？？
